ui/app/components/ui/alert-message/alert-message.stories.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import AlertMessage from './alert-message.component'
+
+export default {
+  title: 'StyledMessage',
```

Comment:
I'm a bit confused by how this stuff is named. it's called a "StyledMessage" in Storybook, but the component is called "alert-message". Yet this type of message is called a "toast" in the Figma designs, and "alert" is used to describe something else.

Could we try to be consistent with the naming - maybe call this a "toast-message"? :thinking:

Reply:
Name changed as suggested in 5b7967969